src/wallet/test/coinselector_tests.cpp

```diff
@@ -345,6 +348,7 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
 
         CoinsResult available_coins;
 
+        coin_selection_params_bnb.m_effective_feerate = CFeeRate(0);
```

Comment:
Nit: This test was already a bit weird to me, because it is categorized as belonging to "BnB", but the composition of preset inputs and the selection of additional inputs is a `SelectCoins`-level functionality that has little to do with specific coin selection algorithms. Composability should be tested at `SelectCoins` level (which it is in `spend_tests` and `wallet_fundrawtransaction.py`, although it could probably be more extensive). Since we already tested whether we can select two UTXOs that exactly match an amount in absence of fees above, I expect that we could remove this test altogether without losing coverage.